app/bundles/CoreBundle/Translations/en_US/messages.ini

```diff
@@ -228,6 +228,12 @@ mautic.core.ip_lookup.last_updated="Last updated on %date%"
 mautic.core.ipaddress="IP address"
 mautic.core.label="Label"
 mautic.core.language="Language"
+mautic.core.language.en="English"
+mautic.core.language.en_GB="English (Pommy)"
+mautic.core.language.en_US="English (USA)"
+mautic.core.language.fr="French"
+mautic.core.language.de="German"
+mautic.core.language.tr="Turkish"
```

Comment:
This is an example list, which can be filled out if this seems a "good thing". The translators that are making the ``fr/message.ini`` ``de/messages.ini`` and so on can then put these keys with strings that are the names of the languages from their point of view "Francais", "Allemand", "Deutsche" etc.

Maybe there is somewhere that already has a matrix lookup table of every known language name from the point of view of every other language?